include/z64save.h

```diff
@@ -1282,16 +1282,16 @@ typedef enum {
 // Also related to moon child
 #define WEEKEVENTREG_84_20 PACK_WEEKEVENTREG_FLAG(84, 0x20)
 
-#define WEEKEVENTREG_84_40 PACK_WEEKEVENTREG_FLAG(84, 0x40)
+#define WEEKEVENTREG_RECEIVED_RED_POTION_FOR_KOUME PACK_WEEKEVENTREG_FLAG(84, 0x40)
 #define WEEKEVENTREG_84_80 PACK_WEEKEVENTREG_FLAG(84, 0x80)
 #define WEEKEVENTREG_85_01 PACK_WEEKEVENTREG_FLAG(85, 0x01)
 
 // Unconfirmed: "Bombers Hide & Seek in Progress"
 #define WEEKEVENTREG_85_02 PACK_WEEKEVENTREG_FLAG(85, 0x02)
 
 #define WEEKEVENTREG_85_04 PACK_WEEKEVENTREG_FLAG(85, 0x04)
-#define WEEKEVENTREG_85_08 PACK_WEEKEVENTREG_FLAG(85, 0x08)
-#define WEEKEVENTREG_85_10 PACK_WEEKEVENTREG_FLAG(85, 0x10)
+#define WEEKEVENTREG_RECEIVED_RED_POTION_FOR_KOUME_FAILED_SHOP PACK_WEEKEVENTREG_FLAG(85, 0x08)
+#define WEEKEVENTREG_RECEIVED_RED_POTION_FOR_KOUME_FAILED_WOODS PACK_WEEKEVENTREG_FLAG(85, 0x10)
```

Comment:
Could you explain these two flags a bit more? I'm not quite following

Reply:
These get set if Kotake tries to give the player the red potion for Koume, however all of the player's bottles are full and so unable to receive it. One gets set for when Kotake is in the shop, and the other gets get if this occurs in the woods. Basically if one of these is set in the appropriate location, while the player still hasn't received the potion, each time the player talks to Kotake she tries to give the potion, and if it still fails ends the conversation and prevents the player from shopping.

Reply:
I see, thanks for explaining. I think `FAILED` and `RECEIVED` should be closer together? so:
`WEEKEVENTREG_RECEIVED_FAILED_RED_POTION_FOR_KOUME_IN_SHOP`
`WEEKEVENTREG_RECEIVED_FAILED_RED_POTION_FOR_KOUME_IN_WOODS`
Maybe? I think adding a comment above to explain might help too

Reply:
Sure, though I might just reverse `RECEIVED` and `FAILED` (`WEEKEVENTREG_FAILED_RECEIVED_RED_POTION_FOR_KOUME_IN_SHOP`). I had originally wanted to keep it in the same order as the actual received weekeventreg, but thinking over it again having failed first seems good as it is the important part.